src/data.cc

```diff
@@ -14,11 +14,14 @@
 #include "data/libfm_parser.h"
 #include "data/csv_parser.h"
 
+// To help C Preprocessor with processing c++ templated types
+#define __DMLC_COMMA ,
```

Comment:
Isn't this already defined in data.h which is included in this file

Reply:
Done